app/src/main/res/values/strings.xml

```diff
@@ -134,4 +134,9 @@
 
     <!-- Elevation graph -->
     <string name="elevation_graph_loading_label">Loading graph…</string>
+    <string name="an_error_occurred_while_fetching_the_profile">An error occurred while fetching the profile.</string>
+    <string name="an_error_occurred_while_updating_the_profile">An error occurred while updating the profile.</string>
+    <string name="an_error_occurred_while_deleting_the_profile">An error occurred while deleting the profile</string>
```

Comment:
Nitpick: could you move them into their correct section?

Reply:
Done!